back/db/migrate/20220531123916_create_home_pages.rb

```diff
@@ -9,7 +9,7 @@ def change
       t.boolean :bottom_info_section_enabled, default: false, null: false
       t.jsonb :bottom_info_section_multiloc, default: {}, null: false
 
-      t.boolean :events_enabled, default: false, null: false
+      t.boolean :events_widget, default: false, null: false
```

Comment:
I'm editing the original migration file here, rather than adding a new migration, since we will be merging to the feature branch which has not yet been merged to master.

Reply:
Not any more! Switched to using a new migration to make the changes.